HeterogeneousCore/AlpakaTest/test/reader.py

```diff
@@ -0,0 +1,29 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('Reader')
+
+process.source = cms.Source('PoolSource',
+    fileNames = cms.untracked.vstring('file:test.root')
+)
+
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.MessageLogger.TestAlpakaAnalyzer = cms.untracked.PSet()
+#process.MessageLogger.PortableCollection = cms.untracked.PSet()
+#process.MessageLogger.TestSoA = cms.untracked.PSet()
+
+process.AlpakaServiceSerialSync = cms.Service('AlpakaServiceSerialSync')
```

Comment:
The `TestAlpakaAnalyzer` does not strictly require the `AlpakaServiceSerialSync`, right? If that is really the case, I'd suggest to remove it from the configuration to ensure that such dependence would not creep in.

Reply:
Good point, I've removed it.